### PR TITLE
Remove Question Marks in Admin Form + add Description to Admin Screen

### PR DESCRIPTION
**Pull request by Aashirbadd**
Overview
----------------------------------------
Relating to the issue in: https://www.drupal.org/project/webform_civicrm/issues/3228229

Before
----------------------------------------
Screen Shot 2021-08-17 at 4 39 31 PM

After
----------------------------------------
Screen Shot 2021-08-17 at 4 40 05 PM

Comments
----------------------------------------
Me and @aaayyuusshh will work on adding the icons beside the select elements next!

### CODE REVIEW

File: src/AdminForm.php [hunk 1]
```diff
@@ -1369,6 +1371,7 @@ private function buildOptionsTab() {
       }
       $this->form['additional_options']['confirm_subscription']['#description'] .= implode(', ', $ml) . '</em>';
     }
+
     else {
```
Comment (KarinG) [inline src/AdminForm.php]: Be careful adding lines/space -> best to delete this extra line here.

Comment (Aashirbadd) [reply src/AdminForm.php]: @KarinG Sounds good, I'll remove that extra line!

File: src/AdminForm.php [hunk 2]
```diff
@@ -427,18 +427,18 @@ private function buildContactTab($n, $c) {
       }
     }
   }
-
+fa-id-badge
   /**
```
Comment (KarinG) [inline src/AdminForm.php]: This line/note is what's currently failing your test 🙂

Comment (Aashirbadd) [reply src/AdminForm.php]: I'll have that changed!

**Comment by aaayyuusshh** [comment]
@Aashirbadd Hey, let's get to updating the outdated description next (:

**Comment by Aashirbadd** [comment]
@KarinG I added spaces where there are break tags (new lines) for better readability, however I can change it back if needed!

**Comment by Aashirbadd** [comment]
We seemed to have hit an issue with the icons, so we just focused on removing the question marks and updating the description. The icons should be replaced in a separate PR.

**Comment by KarinG** [comment]
We just can't get any class into the HTML! Looking further into that.

**Pull request merged**